### ISSUE TITLE
change the vote keys

### ISSUE THREAD

**Issue by mugg91** [transhumandesign/kag-base#659]
Vote keys are O for yes, P for no.

These keys are right next to each other so it's possible to press the opposite of what you wanted by mistake.

Change the keys so they are further apart.

**Comment by eps0003** [comment]
I disagree. Actions that directly relate to each other should have keys that are close together. This is the case with admin freeze/unfreeze keys. And I think it's the player's fault for pressing the wrong key by mistake. You shouldn't be voting quickly when you know you only have one chance to vote.

**Comment by bunniewormy** [comment]
make it possible to cancel vote maybe or just change your vote?

**Comment by Snikerss** [comment]
need make hotkeys in settings and if for anybody vote with O & P will problem he can change it

**Comment by Harrison-Miller** [comment]
Yeah, probably just add key binding for it. Also most games I've seen use page up/down for voting.
